.github/workflows/ubuntu.yml

```diff
@@ -18,10 +18,20 @@ jobs:
         with:
           ruby-version: ${{ matrix.ruby }}
           architecture: 'x64'
+      - name: install dependcnies
+        run: sudo apt-get -y --no-install-recommends install libreoffice
       - name: Update gems
         run: |
           gem install bundler -v "~> 2"
           bundle install --jobs 4 --retry 3
       - name: Run specs
         run: |
           bundle exec rake
+      - name: Test reverse_adoc binary
```

Comment:
The same as above may be better to keep them as ruby tests

Reply:
Done, moved it